packages/api/src/hooks/useOnfido.ts

```diff
@@ -104,8 +104,8 @@ const useOnfido = (country?: string, selectedDocument?: string) => {
                 containerId: onfidoContainerId,
                 language: {
                     locale: i18NLanguage,
-                    phrases: ONFIDO_PHRASES,
-                    mobilePhrases: ONFIDO_PHRASES,
+                    phrases: ONFIDO_PHRASES(),
```

Comment:
we are migrating everything to api-v2 package. Please check with @wojciech-deriv on this 🙏

Reply:
Sure will ask @wojciech-deriv to help review this code as well 🙏

Reply:
hey, good catch, 
basically - /packages/api shouldn't be used in legacy code - https://deriv-group.slack.com/archives/C3AHMSM3P/p1697632878910689

we've discusses that with @suisin-deriv , options are:
a) migrate onfido hooks to packages/hooks (probably too much work especially given upcoming accounts-v2)
b) implement feature within component itself, without /packages/api changes
c) implement changes in a way which is compatybile with /packages/api-v2 and apply changes in both /api and /api-v2

Reply:
Thanks @wojciech-deriv will go with `b` approach for this 🙏